core/src/main/java/nu/yona/server/analysis/service/IntervalActivityDTO.java

```diff
@@ -0,0 +1,55 @@
+package nu.yona.server.analysis.service;
+
+import java.util.Date;
+import java.util.Set;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import nu.yona.server.analysis.entities.IntervalActivity;
+import nu.yona.server.goals.service.GoalDTO;
+
+@JsonRootName("intervalActivity")
+public class IntervalActivityDTO
+{
+	private GoalDTO goal;
+	private Date startTime;
+	private Date endTime;
+	private Set<ActivityDTO> activities;
```

Comment:
This DTO is not used yet, but will be used as soon as the activity controller is created. The activity controller should return sub activities if they are not aggregated. Maybe it has to be reworked, so this is a stub.